webapp/impl/src/test/java/cloud/piranha/webapp/impl/DefaultWebApplicationTest.java

```diff
@@ -511,6 +512,31 @@ public void testGetResourcePaths() {
         assertNull(webApp.getResourcePaths("/this_will_be_null/"));
     }
 
+    @Test
+    public void testGetResourcePaths2() {
+        DefaultResourceManager resourceManager = new DefaultResourceManager();
+        DefaultWebApplication webApp = new DefaultWebApplication();
+        webApp.setResourceManager(resourceManager);
+        webApp.addResource(new DirectoryResource("."));
```

Comment:
Please create an exploded webapp in src/test/webapp to use for testing. E.g src/test/webapp/resourcepaths.